core/src/main/java/com/novoda/noplayer/exoplayer/mediasource/ExoPlayerMappedTrackInfo.java

```diff
@@ -0,0 +1,25 @@
+package com.novoda.noplayer.exoplayer.mediasource;
+
+import com.google.android.exoplayer2.source.TrackGroupArray;
+import com.google.android.exoplayer2.trackselection.MappingTrackSelector;
+
+public class ExoPlayerMappedTrackInfo {
```

Comment:
I am asking about this one hehe, Does this one needs to be public? and if so, why?